config/setup/errors.go

```diff
@@ -21,12 +23,17 @@ func Errors(c *Controller) (middleware.Middleware, error) {
 	// Open the log file for writing when the server starts
 	c.Startup = append(c.Startup, func() error {
 		var err error
-		var file *os.File
+		var file io.Writer
 
 		if handler.LogFile == "stdout" {
 			file = os.Stdout
 		} else if handler.LogFile == "stderr" {
 			file = os.Stderr
+		} else if handler.LogFile == "syslog" {
+			file, err = gsyslog.NewLogger(gsyslog.LOG_ERR, "SYSLOG", "caddy")
```

Comment:
Is there any differences between all the LOCALs? If not, let's just default to LOCAL0.